addons/repo-proxy/common/src/main/data/default-rule.groovy

```diff
@@ -21,15 +21,16 @@ import org.commonjava.indy.model.core.*;
 class DefaultRule extends AbstractProxyRepoCreateRule {
     @Override
     boolean matches(StoreKey storeKey) {
-        return "maven".equals(storeKey.getPackageType()) && (StoreType.group == storeKey.getType() || StoreType.hosted == storeKey.getType())
+        return StoreType.group == storeKey.getType() || StoreType.hosted == storeKey.getType()
     }
 
     @Override
     RemoteRepository createRemote(StoreKey key) {
         def pkgType = key.getPackageType()
         def type = key.getType().singularEndpointName()
         def name = key.getName()
-        return new RemoteRepository(key.getPackageType(), key.getName(), String.format("http://some.indy/api/content/%s/%s/%s", pkgType, type, name))
+        return new RemoteRepository(pkgType, String.format("%s-%s", type, name), String.format("http://some.indy/api/content/%s/%s/%s", pkgType, type, name))
```

Comment:
I wonder if there's a way to make the target Indy instance configurable so we don't have to edit a groovy script in order to deploy Indy as a static proxy?

The Groovy script support we have allows you to specify environment stuff, I believe...and if you're implementing an interface I think it will also allow you to @Inject things (though I might be wrong about that). If we can do one of these things, we could bring in the indy target url from a config parameter and avoid the need to modify this script for each deployment.

Reply:
If the target indy is configurable, we might not need the groovy? If the createRemote logic is not going to change, maybe just hard code it.

Reply:
I'd be ok with this for now, and we can change to using the groovy approach that we use in koji or elsewhere if we needed it.